Source/SwiftLintFramework/Models/Example.swift

```diff
@@ -3,22 +3,36 @@
 public struct Example {
     /// The contents of the example
     public private(set) var code: String
+    /// The untyped configuration to apply to the rule, if deviating from the default configuration
+    public private(set) var configuration: Any?
```

Comment:
Why is the configuration untyped? Might be nice to include something in the comma, so others understand the intended usage.